plugin/index.tpl.html

```diff
@@ -29,6 +29,7 @@ <h1>
 		</div>
 
 		<meta property="repo" />
+		<meta property="url" />
```

Comment:
We should probably actually display these in the plugin listing page instead of having them as hidden `<meta>`s here too.

Reply:
Totally agree. When I added my first plugin, I was confused about why the repo field is here: should I duplicate info from the plugins listing page here or what?